### PR TITLE
*: cut v0.53.0

### PR DESCRIPTION
**Pull request by simonpasquier**
## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
+## 0.53.0 / 2021-12-15
```
Comment (paulfantom) [inline CHANGELOG.md:1]: The https://github.com/prometheus-operator/prometheus-operator/pull/4323 is missing.

Do we also want to mention that default repo branch was changed in https://github.com/prometheus-operator/prometheus-operator/pull/4384 ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+
+* [CHANGE] Added startup probe to Prometheus. #4433 #4369
+* [FEATURE] Added support for mute time intervals in the AlertManagerConfig CRD. #4388
+* [FEATURE] Added support for new matching syntax in the routes configuration of the AlertmanagerConfig CRD. #4332
+* [FEATURE] Added support for new matching syntax in the inhibit rules configuration of the AlertmanagerConfig CRD. #4329
+* [FEATURE] Added `retryOnRateLimit` in the Remote Write configuration of the Prometheus CRD. #4420
+* [FEATURE] Added support for PagerDuty links and images in the AlertManagerConfig CR. #4425
+* [ENHANCEMENT] Optimized the generated Prometheus configuration to make it compatible with the new Prometheus agent mode. #4417
+* [BUGFIX] Improved validation for the Alertmanager CRD to match with upstream Alertmanager. #4434
+* [BUGFIX] Propagated annotations from spec.podMetadata to the StatefulSet's pods. #4422
```
Comment (paulfantom) [inline CHANGELOG.md:11]: nit:
```suggestion
* [BUGFIX] Fixed propagation of annotations from spec.podMetadata to the StatefulSet's pods. #4422
```

File: CHANGELOG.md (hunk 1 above)
Comment (simonpasquier) [reply CHANGELOG.md]: done for #4323 thanks! I don't feel like the changelog should mention the default branch change since it has no effect on users?

**Review by paulfantom — APPROVED** [review]
lgtm

**Pull request merged**